Source/Scene/BatchTable.js

```diff
@@ -273,7 +275,12 @@ define([
         return Cartesian4.fromElements(x, y, z, w, result);
     }
 
-    var scratchFloatArray = new Float32Array(1);
+    var scratchFloatArray;
+    if (FeatureDetection.supportsTypedArrays()) {
+        scratchFloatArray = new Float32Array(1);
+    } else {
```

Comment:
Since Cesium doesn't work without typed arrays, we normally just return immediately.  So I would change this to:

```javascript
if (!FeatureDetection.supportsTypedArrays()) {
    return;
} 

var scratchFloatArray = new Float32Array(1);
```